doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst

```diff
@@ -676,6 +676,86 @@ The following changes are recommended for your application to work optimally aft
   This change enables the use of standard |NCS| tools and procedures for building and configuring the controller.
   The SoftDevice Controller will for some time not have a QDID for the ISO operation.
 
+* For Mesh applications, the Sensor API has been re-written, and the previous Sensor API is deprecated.
+  A new Sensor API has been created that should be used for new applications and applications that will be continuously updated.
+  The new Sensor API is described in :ref:`bt_mesh_sensors_readme`.
+  Usage is shown in :ref:`bluetooth_mesh_sensor_client`, :ref:`bluetooth_mesh_sensor_server` and :ref:`bluetooth_mesh_light_lc` samples, as they all have been updated to use the new Sensor API.
+
+  The :kconfig:option:`CONFIG_BT_MESH_SENSOR_USE_LEGACY_SENSOR_VALUE` is set default to `=y`.
+  This means for existing applications there is no need to do anything to continue using the deprecated Sensor API.
+  The application will get a deprecation warning when compiled, that the user can choose to disregard.
+  This will work until the deprecation period is over for the deprecated Sensor API.
+  When the deprecation period is over the deprecated Sensor API will be removed, and the application will no longer compile if not updated to the new Sensor API.
+
+  New and existing applications that want to use the new Sensor APIs must set :kconfig:option:`CONFIG_BT_MESH_SENSOR_USE_LEGACY_SENSOR_VALUE` to `=n` in their configuration.
+  This configuration value will be removed when the deprecation periode is over, and the configuration value then need to be removed from the application configuration.
+  It is important to check if :kconfig:option:`CONFIG_FPU` is set to `=y` to support accelerated floating point operations, and possible add :kconfig:option:`CONFIG_CBPRINTF_FP_SUPPORT` to support floating point printing.
```

Comment:
```suggestion
  To use the new sensor API for new and existing samples and applications, disable the Kconfig option :kconfig:option:`CONFIG_BT_MESH_SENSOR_USE_LEGACY_SENSOR_VALUE` in the configuration.
  This configuration option will be removed when the deprecation period is over, and then it has to be removed from the sample and application configuration.
  It is also recommended to enable the Kconfig option :kconfig:option:`CONFIG_FPU` to support the accelerated floating point operations, and the Kconfig option :kconfig:option:`CONFIG_CBPRINTF_FP_SUPPORT` to support the  floating point printing.
```

Reply:
I have a question. Is it correct to add "samples and applications" instead of just "applications". The samples using sensor API are updated, and the users should not migrate samples. So why add samples?